cluster/log-dump/log-dump.sh

```diff
@@ -136,25 +137,25 @@ function copy-logs-from-node() {
 function save-logs() {
     local -r node_name="${1}"
     local -r dir="${2}"
-    local files="${3}"
+    local save_files="${3}"
```

Comment:
Quotes are superfluous when assigning from other variable.
You can test that e.g. by typing:

$ a='foo bar'
$ b=$a
$ echo $b
foo bar
$